docs/sources/get-started/quick-start.md

```diff
@@ -326,9 +326,9 @@ You have completed the Loki Quickstart demo. So where to go next?
 {{< docs/ignore >}}
 ## Back to docs
 Head back to where you started from to continue with the Loki documentation: [Loki documentation](https://grafana.com/docs/loki/latest/get-started/quick-start/).
-{{< /docs/ignore >}}
 
 ## Complete metrics, logs, traces, and profiling example
+{{< /docs/ignore >}}
```

Comment:
I think that rather than move the Killercoda markdown (docs/ignore), what we want to do here is just delete the second heading (line 330).

Reply:
LGTM, DONE